### PR TITLE
fix(coinex) - skip cs <Q

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: skip-tests.json [hunk 1]
```diff
@@ -644,6 +644,7 @@
     },
     "coinex": {
         "skipWs": "timeouts",
+        "skipCSharp": "service busy: https://app.travis-ci.com/github/ccxt/ccxt/builds/269539760#L3513",
```
Comment (carlosmiei) [inline skip-tests.json:647]: why only c# then?

Comment (ttodua) [reply skip-tests.json:647]: i see such scenarios more frequent, when the exchange connection timeouts/fails in many times only in c#, but not in others. so I just suppose there might be some underlying lib/header/etc for c#, as it's relatively newer and there might be somethings in the backgrounds. however, as a quick workaround, we can only skip in c# for now and observe what happens.

Comment (ttodua) [reply skip-tests.json:647]: here another example: https://app.travis-ci.com/github/ccxt/ccxt/builds/269575006#L3639

**Comment by carlosmiei** [comment]
I wouldn't do this right until, until we figure out exactly what's wrong with the c# version

**Comment by ttodua** [comment]
but i dont undersatnd what's wrong if we skip some 3rd party exchange and it would avoid travis breaks, so, "fix" could happen paralelly.
but oky, as you think

**Comment by carlosmiei** [comment]
@ttodua coinex is partner btw

**Pull request closed without merging**